src/components/data/DataTable.js

```diff
@@ -359,9 +359,9 @@ const DataTable = props => {
                       }, [
                         h(HeaderCell, [
                           !!columnNamespace && span({ style: { fontStyle: 'italic', color: colors.dark(0.75), paddingRight: '0.2rem' } },
```

Comment:
The exact style (i.e. `{ style: { fontStyle: 'italic', color: colors.dark(0.75), paddingRight: '0.2rem' } }`) is used in `UploadPreviewTable` too. I wonder if it's worth factoring it out so it's a bit more difficult to be inconsistent.